gaia-radio.js

```diff
@@ -0,0 +1,239 @@
+/*globals define*/
```

Comment:
This is looking like a new file, should this have been `git mv`ed instead? Otherwise we'll lose `git blame` history.